docs/cow-protocol/reference/core/definitions.md

```diff
@@ -20,10 +20,20 @@ A single order submitted to the CoW Protocol API (i.e. `GPv2Order.Data`)
 
 An intent by a smart-contract to place `0..n` discrete orders under some programmatic logic. For example, a conditional order may only be valid if the price of an asset reaches a certain level.
 
+#### Limit Order
+An order to swap specified `sellAmount` of `sellToken` for a specified `buyAmount` of `buyToken` or better.
+
+#### TWAP Order
+Time-Weighted Average Price order. It is an algorithmic trade execution strategy that aims to achieve an average execution price close to the time-weighted average price of the specified period.
```

Comment:
```suggestion
```

In general, I don't think we should define "external" definitions (ie. non-specific to CoW Protocol / batch auctions). The unique thing about CoW is it gives surplus over limit orders, which is defined below in _surplus_. "Limit order" and "TWAP order" are known terms (or able to be googled). We should however IMO on the "Limit order" and/or "TWAP order" page within the concepts section link to a wikipedia article that is able to source the external definition.